frontend/src/components/Comment/CommentAPI.js

```diff
@@ -0,0 +1,34 @@
+import Axios from 'axios';
+import '../../config';
+
+export const getCommentByPicture = async (pictureId) => {
```

Comment:
Is it necessary to add `async` since we're returning the Promise anyway and we're not using the await keyword.